app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -295,21 +293,23 @@ div.picture_thumbnail div.picture_image {
   }
 }
 
-div#cells {
+#cells {
   min-height: 100px;
 }
 
 .linkable_essence_tools {
   position: absolute;
-  bottom: 0;
+  bottom: 6px;
   right: 0;
 
   .icon_button {
     @include button-defaults;
+    width: 29px;
```

Comment:
Properties should be ordered bottom, height, margin, padding, position, right, width